### ISSUE TITLE
Allow editing all bookmark details

### ISSUE THREAD

**Issue by tahutempe** [karakeep-app/karakeep#375]
Not sure if i missed this. I can't seem to find a way to edit the original URL.

**Comment by MohamedBassem** [comment]
What do you expect to happen if you change the URL to a completely different website? Do you expect hoarder to go and recrawl the new URL?

**Comment by tahutempe** [comment]
There are two things that I expect to see:
- View the original full URL - maybe include "copy" function.
- Edit the URL. After editing the URL, the hoarder should go and recrawl it.

Thanks and really appreciate for creating this tool.

**Comment by kiinami** [comment]
I also would love to have this feature. I have imported links from other services and some have changed domains so the crawler fails to get the content. I'm right now creating new bookmarks manually, but it is inefficient, since I have to search for the failed ones by hand, and the new bookmark does not have the original date in which it was saved. Not the worst, but still.

Thank you for this wonderful app, btw, it has saved me days of tagging my expansive collection.

**Comment by kbhbhk** [comment]
> * View the original full URL - maybe include "copy" function.
> * Edit the URL. After editing the URL, the hoarder should go and recrawl it.

Exactly! Second this both idea strongly. It's hard to see the full URL at a glance right now.
Thanks for this amazing project btw!

**Comment by cvladan** [comment]
Please add this feature. See #970

**Comment by bektaskemal** [comment]
It should at least be supported for broken links. Usually a small change can fix the link.

**Comment by huyz** [comment]
I think that recrawling should be optional on a link-by-link basis. There's a use case for both.
Sometimes the original link and archive are no good.
Sometimes the URL just needs to be cleaned up (remove marketing junk) or expanded or canonicalized.

Maybe once you edit the URL, a dialog should come up and ask "re-crawl?". And if people care, then there could be a checkbox for "always".
As for me I'll always want to decide each time

**Comment by MohamedBassem** [comment]
This is coming in the next release: 3207264fc13c275d6dcfbd2628cc6b3974ceeaed
